docs/subtensor-nodes/subtensor-storage-query-examples.md

```diff
@@ -552,7 +552,7 @@ title: "Subtensor Storage Query Examples"
 
     substrate = SubstrateInterface(url="ws://127.0.0.1:9945")
     netuid = 1
-    result = substrate.query('SubtensorModule', 'LastMechanismStepBlock', [netuid])
+    result = substrate.query('SubtensorModule', 'LastMechansimStepBlock', [netuid])  # ..nsim [sic] typo on the chain
```

Comment:
What is ".. nsim [sic]" in the comment? No one understands what this is. Either remove the entire comment or simply say "Typo on the chain". Same comment for both the commits.

Reply:
I alerted the dev and they are going to fix it, but we don't have ETA. So having a simple comment like "Typo on the chain" will be useful indeed.

Reply:
https://en.wikipedia.org/wiki/Sic [sic] is a comment to point out that an error is quoted as is. Many people do understand what it means and it is a correct way to mark out quoted erroneous text used in many languages